pkg/pillar/types/globalconfigold.go

```diff
@@ -0,0 +1,356 @@
+// Copyright (c) 2018 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Can we add an XXX comment that this file should be removed when all 4.X and 5.0 versions of EVE are gone?

Reply:
Yeah.. Wanted to add that.. Done..